lib/index.js

```diff
@@ -68,7 +68,7 @@ const PERMISSIONS = bedrock.config.permission.permissions;
 api.set = async ({
   actor, account, email, type, clientId, serviceId, hash,
   authenticationMethod = type, requiredAuthenticationMethods = [],
-  notify = true
+  notify = true, entryStyle
```

Comment:
set `entryStyle` with a default here as well.  There is also jsdocs here that need to be updated.

Reply:
I see also, that this `entryStyle` will only apply to the `nonce` type.  So this should probably be `nonceEntryStyle`

Reply:
@mattcollier -- We could have `typeOptions` (or something along those lines) such that are token type specific and use `entryStyle` within those. That would enable more future flexibility, I'd think.